lib/expand/join.go

```diff
@@ -763,7 +762,15 @@ type operationContext struct {
 // For a local gravity cluster, it will attempt to start the expand operation
 // and will return an operation context wrapping a new expand operation.
 func (p *Peer) connectLoop() (*operationContext, error) {
-	ticker := backoff.NewTicker(leader.NewUnlimitedExponentialBackOff())
+
+	// Lots of joining nodes create load on the gravity-site controller cycling on creating join operations
+	// Set a high maximum so lots of queued joins don't create too much load
+	b := backoff.NewExponentialBackOff()
```

Comment:
I removed NewUnlimitedExponentialBackoff because the parameters it sets are not appropriate for this fall back, and the abstraction hide the details of what's actually being configured.

Reply:
Not sure I follow. Here's the implementation of `NewUnlimitedExponentialBackoff`:
```go
func NewUnlimitedExponentialBackOff() *backoff.ExponentialBackOff {
	b := backoff.NewExponentialBackOff()
	b.MaxElapsedTime = 0
	return b
}
```
and can be used to set any additional parameters:
```go
b := utils.NewUnlimitedExponentialBackoff()
b.Multiplier = 2
b.MaxInterval = time.Minute
```
my concern was specifically with `b.MaxElapsedTime = 0` which is not as clear as `UnlimitedExponentialBackoff`

Reply:
The previous function must've had a different name, but it just buried the parameters given to the backoff. So I find it non intuitive to wrap everything because now you need added context of what some random utility function does.